### ISSUE TITLE
Exporter not working properly on live

### ISSUE THREAD

**Issue by Waseem-Jutt** [rap2hpoutre/fast-excel#47]
Error:
syntax error, unexpected '?'
--
in Importable.php (line 57)
>>>>>>>>>>>>>>>>>>>>>>>>>>

`    public function export(Request $request) {

        if ($request->s_search) {
                $timelines = Timeline::where([ ['type', 'user'], ['name', 'like', '%'.$request->s_name.'%'] ])->join('users', 'users.timeline_id', '=', 'timelines.id')
                ->where([['deg_year', 'like', '%'.$request->s_year.'%'],['deg_ach', 'like', '%'.$request->programme_id.'%'],['school', 'like', '%'.$request->s_sname.'%']])->get();
        } else {
            $timelines = Timeline::where('type', 'user')->get();
        }
        // echo '<pre>';
        // print_r($timelines);
        // echo '</pre>';
        // die;
        
        $xlxs = rand().'_Users';
        (new FastExcel($timelines))->export('/export/'.$xlxs.'.xlsx', function ($timeline) {
            return [
                'Email'    => $timeline->user->email,
                'username' => $timeline->username,
            ];
        });   
        //return (new FastExcel($timelines))->download('./export/'.$xlxs.'.xlsx');
         
         
         return redirect('/admin/users?'.$request->getQueryString());
    }`

**Comment by rap2hpoutre** [comment]
Your PHP version seems to have a problem with this line: https://github.com/rap2hpoutre/fast-excel/blob/a00433fbb4d56bf834ae1aa7ac567981d3bdb83a/src/Importable.php#L57

What is your PHP version? You need at least version 7 of PHP to make this library work. The php 7.0 feature that does not work is the null coalesce operator: http://php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op

TL;DR: upgrade to PHP 7 (you might upgrade soon, because 5 versions are expiring in few month: http://php.net/supported-versions.php)

**Comment by Waseem-Jutt** [comment]
My php version was most latest 7.2.. But this still not worked and i changed it with some core php code.. and that working nicely.

**Comment by darshanadaptit** [comment]
> My php version was most latest 7.2.. But this still not worked and i changed it with some core php code.. and that working nicely.

I am fetching same issue could you please me know what you tweaked in code
